lib/response.js

```diff
@@ -425,6 +425,33 @@ module.exports = {
   },
 
   /**
+   * Append additional header `field` with value `val`.
+   *
+   * Examples:
+   *
+   *    this.append('Link', ['<http://localhost/>', '<http://localhost:3000/>']);
+   *    this.append('Set-Cookie', 'foo=bar; Path=/; HttpOnly');
+   *    this.append('Warning', '199 Miscellaneous warning');
+   *
+   * @param {String} field
+   * @param {String|Array} val
+   * @api public
+   */
+
+  append: function(field, val){
+    var prev = this.get(field);
+    var value = val;
+    if (prev) {
+      // concat the new and prev vals
+      value = Array.isArray(prev) ? prev.concat(val)
+        : Array.isArray(val) ? [prev].concat(val)
+        : [prev, val];
+    }
+
+    return this.set(field, value);
+  },
```

Comment:
Oh, it's a direct port :) I'd say that's why I choose to use an intermediate :) Just because I didn't want to be sure it was a per hit, haha.